File: algorithm/src/main/java/baekjoon/p05565/Main.java
package baekjoon.p05565;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] inputs = new int[10];

        for (int i = 0; i < inputs.length; i++) {
            inputs[i] = sc.nextInt();
        }

        System.out.println(solve(inputs));
    }

    private static int solve(int[] inputs) {
        int total = inputs[0];
        int sum = 0;

        for (int i = 1; i < inputs.length; i++) {
            sum += inputs[i];
        }

        return (total - sum);
    }

}

File: algorithm/src/main/java/baekjoon/p05565/Main.kt
package baekjoon.p05565

fun main(args: Array<String>) {
    val inputs = IntArray(10)

    for (i in 0 until inputs.size) {
        inputs[i] = readLine()!!.toInt()
    }

    println(solve(inputs))
}

private fun solve(inputs: IntArray): Int {
    val total = inputs[0]
    var sum = 0

    for (i in 1 until inputs.size) {
        sum += inputs[i]
    }

    return (total - sum)
}